src/azure-cli/azure/cli/command_modules/config/_params.py

```diff
@@ -28,3 +28,13 @@ def load_arguments(self, _):
         c.argument('local', action='store_true',
                    help='Include local configuration. Scan from the working directory up to the root drive, then the global configuration '
                         'and unset the first occurrence.')
+
+    with self.argument_context('config parampersist show') as c:
+        c.positional('name', nargs='*', help='Space-separated list of parameter persistence names.')
+
+    with self.argument_context('config parampersist delete') as c:
+        c.positional('name', nargs='*', help='Space-separated list of parameter persistence names. Either positional name argument or --all can be specified.')
```

Comment:
should we use `+`  to prevent no value added for the parmater?

Reply:
Support no value here.